test/collections.js

```diff
@@ -563,6 +563,8 @@
     equal(3, _.max([1, 2, 3, 'test']), 'Finds correct max in array starting with num and containing a NaN');
     equal(3, _.max(['test', 1, 2, 3]), 'Finds correct max in array starting with NaN');
 
+    equal([3, 6], _.map([[1, 2, 3], [4, 5, 6]], _.max), 'Finds correct max in array when mapping through multiple arrays');
```

Comment:
I will switch the test to deepEqual, commit, and then include in my new commit message that it does not work when passed an array of objects.

After the commit is merged then we can annotate the index.html to reflect this change. I gathered from reading the contributions guidelines that we should merge changes and then change index.html. I assume we are following that procedure here?